vertx-hazelcast/src/test/java/org/vertx/java/spi/cluster/impl/hazelcast/HazelcastAsyncMultiMapTest.java

```diff
@@ -0,0 +1,156 @@
+package org.vertx.java.spi.cluster.impl.hazelcast;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.JoinConfig;
+import com.hazelcast.config.NetworkConfig;
+import com.hazelcast.core.*;
+import com.hazelcast.instance.GroupProperties;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.vertx.java.core.AsyncResult;
+import org.vertx.java.core.Handler;
+import org.vertx.java.core.impl.DefaultFutureResult;
+import org.vertx.java.core.impl.DefaultVertx;
+import org.vertx.java.core.spi.Action;
+import org.vertx.java.core.spi.cluster.ChoosableIterable;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static com.hazelcast.instance.HazelcastTestUtil.closeConnectionBetween;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author <a href="mailto:andreas.berger@kiwigrid.com">Andreas Berger</a>
+ */
+public class HazelcastAsyncMultiMapTest {
```

Comment:
It seems you're only testing that hz multimap does not merge "correctly". Shouldn't you test for the clearance of the cache, should you? I thought clearing the cache is what this PR is all about.